zio-http/src/test/scala/zio/http/TestServer.scala

```diff
@@ -7,22 +7,17 @@ import zio.http.Server.ErrorCallback
  * Enables tests that make calls against "localhost" with user-specified
  * Behavior/Responses.
  *
- * @param state
- *   State that can be consulted by our behavior PartialFunction
  * @param behavior
  *   Describes how the Server should behave during your test
  * @param driver
  *   The web driver that accepts our Server behavior
  * @param bindPort
  *   Port for HTTP interactions
- * @tparam State
- *   The type of state that will be mutated
  */
-final case class TestServer[State](
```

Comment:
Need to move this to new package so it is accessible to users.

Reply:
Will be done in the next project/packaging bit of work.